src/main/java/com/rcv/GuiConfigController.java

```diff
@@ -60,11 +60,12 @@
 import javafx.stage.FileChooser.ExtensionFilter;
 import javafx.util.StringConverter;
 
+@SuppressWarnings("WeakerAccess")
```

Comment:
what is this?

Reply:
It suppresses future incorrect / unnecessary IntelliJ Code Inspection warnings.